src/Microsoft.VisualStudio.Editor.Razor/VisualStudioRazorParserFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal abstract class VisualStudioRazorParserFactory
```

Comment:
The public one should be factory

Reply:
Hu? There doesn't need to be a public one. Could you expand?

Reply:
I'm not sure this comment makes snes